src/Adapter/Search/SearchProductSearchProvider.php

```diff
@@ -70,10 +73,14 @@ public function runQuery(
             );
             $products = $result['result'];
             $count    = $result['total'];
+
+            Hook::exec('actionSearch', array('expr' => $queryString, 'total' => $count));
```

Comment:
i think that this is a bad practice to use short names for things like this, why not "query", "searched_query" or just "expression"?